content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -9,13 +9,8 @@ menu:
     parent: About the project
 ---
 
-## 1.9.7 [2022-06-06]
-
-{{% warn %}}
-An edge case regression was introduced into this version that may cause a constant build-up of hinted handoff if writes are rejected due to malformed requests. We're reverting back to InfluxDB Enterprise 1.9.6 as the official stable version. If you experience write errors and hinted hand-off growth, we recommend reverting back to 1.9.6 or upgrading to 1.9.8 when released.
-{{% /warn %}}
+## 1.9.8 [2022-07-05]
 
-<!--
 ### Features
 - Expose passive node feature to influxd-ctl and the API.
 - Throttle inter-node data replication, both incoming writes and hinted hand-off, when errors are encountered.
```

Comment:
```suggestion
- Throttle inter-node data replication, both incoming writes and hinted hand-off, when errors are encountered.
- Add new `hh_node` measurement to the `/debug/vars` monitoring telemetry. This tracks more accurately with the `max-size` configuration setting for hinted handoff in data nodes.
```

Reply:
Not sure how you are adding updates but a lot of these fixes you are suggesting don't actually need to be made, they look correct in VS Code and render without issue in our mock up server. I did add the hh_node part which was not included in this release notes issue https://github.com/influxdata/docs-v2/issues/4124.